src/main/resources/lib/ssb/utils/variantUtils.ts

```diff
@@ -139,13 +139,32 @@ function calculateQuarter(previousFrom: string, language: string): string {
   const date: Date = new Date(previousFrom)
   const fromMonth: number = date.getMonth() + 1
   const quarter: number = Math.ceil(fromMonth / 3)
+
+  if (language === 'en') {
+    const suffix = getSuffix(quarter)
+    return `figures for ${quarter}${suffix} Quarter ${date.getFullYear()}`
+  }
+
   return localize({
     key: 'period.quarter',
     locale: language,
     values: [quarter.toString(), date.getFullYear().toString()],
```

Comment:
You should instead replace the quarter.toString() string in values in localize when the language is english.

Explanation: 
This is how period.quarter looks like in the phrases_en.properties file = figures for {0}. quarter {1}. The first string in the values array will replace {0}, whilst the second string will replace {1} etc.

Reply:
removed the . after {0}, and added localization for en version